Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This happened because you updated your fork with Master? :)

The pull request should have only updated itself in the event that you had to check in those changes to your branch. Which should not have happened.

I suspect we might have to close this pull and open a new one with only your updates.